components/blitz/src/omero/gateway/model/ShapeSettingsData.java

```diff
@@ -467,27 +473,41 @@ public String getMarkerStart()
      */
     public String getMarkerEnd()
     {
+        Shape shape = (Shape) asIObject();
+        if (shape instanceof Line) {
+            Line l = (Line) shape;
+            return l.getMarkerEnd() != null ? l.getMarkerEnd().getValue()
+                    : "";
+        }
         return "";
     }
 
     /**
-     * Returns the marker start.
+     * Sets the marker start.
      *
      * @param start The value to set.
      */
-    public String setMarkerStart(String start)
+    public void setMarkerStart(String start)
     {
-        return "";
+        Shape shape = (Shape) asIObject();
```

Comment:
I don't think that's necessary, `rtypes.rstring(start)` will return empty string in case of `start == null`.